tasks/package.js

```diff
@@ -18,6 +18,57 @@ var extensionDescriptor = require('./helpers/extensionDescriptor');
 var fs = require('fs');
 var getPaths = require('./helpers/getPackagePaths.js');
 var path = require('path');
+var process = require('process');
+var yargs = require('yargs/yargs')
+var { hideBin } = require('yargs/helpers')
+var argv = yargs(hideBin(process.argv))
+  .usage('Usage: $0 [options]')
+  .options({
+    v: {
+      alias: 'verbose',
+      describe: 'Display extra information, if available',
+      type: 'boolean'
+    },
+    o: {
+      alias: 'out',
+      describe: 'The directory path or file path to write the archive',
+      type: 'string'
+    },
+    'dry-run': {
+      describe: 'Preview how the archive will be built',
+      type: 'boolean'
+    }
+  })
+  .argv
+
+var getOutputLocation = function() {
```

Comment:
For testing purposes, I found it useful to another output directory, and adding this feature resolves #17 .